src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -49,6 +49,14 @@ public AddCommand(String name,
         );
     }
 
+    /**
+     * Checks if commandType is mutatable
```

Comment:
`mutatable` -> `mutable`. Also it is better to be more specific here and says `Returns if the command can potentially mutate the data`